libs/ui/src/diagnostics/mark_scan_readiness_report.tsx

```diff
@@ -49,6 +49,7 @@ export function MarkScanReadinessReportContents(
         {...accessibleControllerProps}
         diagnosticType="mark-scan-accessible-controller"
         title={DiagnosticSectionTitle.AccessibleController}
+        detectedLabel="Connected"
```

Comment:
Why not have them all say "Connected"?

Reply:
It felt wrong for the PAT input because there's a difference between the jack being detected and a sip & puff device being connected. But for everything else that's supposed to be permanently connected, using "Connected" makes sense. Thinking about it now, I agree for simplicity we should just go with "Connected" for all.